Tasks/UseDotNetV2/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "module": "commonjs",
+        "types": ["mocha"],
+        "sourceMap": true,
+        "sourceRoot": "C:\\github\\azure-pipelines-tasks\\tasks\\usedotnetv1\\"
```

Comment:
This looks suspicious to me. Is this supposed to be checked in? Seems user specific

Reply:
We need the tsconfig, but yeah I don't think we need the `sourceRoot`/some of the other options (I think that came when I originally pulled this over from the DotNetCoreInstaller task?). Updated accordingly

Reply:
Does it work to just specify a relative path for sourceRoot? I was mostly concerned about the absolute path. Disclaimer, I havent really developed on this code yet so Im not sure if the dev flow really even uses the source mapping

Reply:
I think we just don't need it here/don't use it in general.